odxtools/table.py

```diff
@@ -93,7 +97,7 @@ def __init__(
 
```

Comment:
hm, I always thought that constructors are not supposed to return any values. even though python does not seem to complain about it, I guess it would be good to remove the return statement...

Reply:
Could you please help me with the line, where you see the return statement?.

Reply:
never mind, I've somehow looked and some ephemeral state of your series. (all is good.)